main.go

```diff
@@ -462,11 +522,32 @@ func main() {
 	td := tdglib.NewToDoGenerator(env.sourceRoot(),
 		includePatterns,
 		excludePatterns,
-		false /*blame*/,
+		env.assignFromBlame,
 		env.minWords,
 		env.minChars,
 		env.concurrency)
 
+	if env.assignFromBlame {
+		root := env.sourceRoot()
+		for strings.HasSuffix(root, "/.") || strings.HasSuffix(root, "/") {
+			root = strings.TrimSuffix(root, "/.")
+			root = strings.TrimSuffix(root, "/")
+		}
+
+		command := "git"
+		args := []string{"config", "--global", "--add", "safe.directory", root}
```

Comment:
You are right. I can't rememebr exactly why, but before I had thought it wouldn't fit right in tdg. But now I think it would fit better in tdg. But you're saying that tdg is already doing this? I don't think so. Will create an MR for it on Gitlab.